Source/NSCoder.m

```diff
@@ -397,6 +408,12 @@ - (id) decodeObjectForKey: (NSString*)aKey
   return nil;
 }
 
+- (id) decodeObjectOfClass: (Class)cls forKey: (NSString *)aKey
```

Comment:
Please also add the method decodeObjectOfClasses:forKey: as that is the more basic one, which should actually be implemented.